src/binary-reader-objdump.cc

```diff
@@ -643,7 +646,8 @@ Result BinaryReaderObjdumpDisassemble::OnEndExpr() {
 
 Result BinaryReaderObjdumpDisassemble::BeginFunctionBody(Index index,
                                                          Offset size) {
-  printf("%06" PRIzx " func[%" PRIindex "]", state->offset, index);
+  current_function_offset = state->offset;
+  printf("%06" PRIzx " func[%02" PRIindex "]", state->offset, index);
```

Comment:
Seems a bit strange to me to use a leading zero for a decimal number. Also, none of the other indexes have leading zeroes.

Reply:
Its for alignment of the instructions which are on the right of this information.